### ISSUE TITLE
Confused about Greatest.h in "Compiling a Lisp: Integers"

### ISSUE THREAD

**Issue by daneelsan** [tekknolagi/tekknolagi.github.com#11]
I'm quite confused about this header. Should we just "use" it, as is? If that is the case then perhaps a link to the file would be OK.

Also, when I compare the definition of the macro `EXPECT_EQUAL_BYTES` in page vs in github:
page:
```c
#define EXPECT_EQUALS_BYTES(buf, arr) ASSERT_MEM_EQ(arr, (buf)->address, sizeof arr)
```
git:
```c
#define EXPECT_EQUALS_BYTES(buf, arr) Testing_expect_buffer_equals_bytes((buf), (arr), sizeof arr)
```

Is this important? I can't get it to compile with `gcc -Wall -Wextra -pedantic -fno-strict-aliasing`

**Comment by tekknolagi** [comment]
Ah, I meant to update that. It should be `ASSERT_MEM_EQ`, since I got rid of `Testing_expect_buffer_equals_bytes`. I discovered the happy existence of `ASSERT_MEM_EQ` in-between posts :)

**Comment by tekknolagi** [comment]
Fixed in 31b9a776be5ef1abfb681923b5341b78c953de20. Let me know if that works for you. If you want, check out this repository and go into `assets/code/lisp`. There should be a Makefile there to compile and test everything (with `make` and `make test`).

**Comment by tekknolagi** [comment]
Going to close pending further response.
